MapleServer2/Tools/ScriptLoader.cs

```diff
@@ -32,7 +32,7 @@ public ScriptLoader(string scriptName, GameSession session = null)
             // Register script manager as an proxy object
             // Documentation: https://www.moonsharp.org/proxy.html
             UserData.RegisterProxyType<ScriptManager, GameSession>(r => new(r));
-            Script.Globals["Helper"] = new ScriptManager(session);
+            Script.Globals["PlayerHelper"] = new ScriptManager(session);
```

Comment:
Maybe? Thinking about it I guess a better name would be like Script manager then we can short it to sm, so it would be something like sm.GetPlayer.JobId? Just like ms1 scripts xD

Reply:
I agree with `sm`